examples/with-google-analytics-typescript/lib/gtags.ts

```diff
@@ -0,0 +1,12 @@
+const GA_TRACKING_ID = process.env.NEXT_PUBLIC_GA_ID ?? ''
+
+// measure PV
+const pageview = (url: string): void => {
+  if (GA_TRACKING_ID === '') return
+
+  window.gtag('config', GA_TRACKING_ID, {
+    page_path: url,
+  })
```

Comment:
IMHO it is not the correct way of doing this.

Check [`gtags.js` documentation](https://developers.google.com/analytics/devguides/collection/gtagjs/pages#manual_pageviews) about how to manually send page view event on SPA.